Source/WebCore/rendering/RenderListBox.h

```diff
@@ -111,8 +111,8 @@ class RenderListBox final : public RenderBlockFlow, public ScrollableArea {
     virtual bool shouldPanScroll() const { return true; }
     void panScroll(const IntPoint&) override;
 
-    int verticalScrollbarWidth() const override;
```

Comment:
We should create a No/Yes enum for this parameter, to make the calling side clearer

`enum class ShouldIncludeGutterForDocumentRenderer : bool { No, Yes }`

Reply:
Removed this.